app/src/main/res/values/strings.xml

```diff
@@ -228,6 +231,7 @@ If you are overwhelmed by the number of quests, you can always fine-tune which q
 
     <string name="confirmation_authorize_now">"You need to authorize with your OSM user account to publish your answers. Authorize now?"</string>
     <string name="confirmation_authorize_now_note2">You can also do this later on the profile screen.</string>
+    <string name="confirmation_permission_upgrade">"To upload a GPS trace, you need to authorize with your OpenStreetMap user account to publish traces. This will log you out of the app and ask you to re-login. Proceed now?"</string>
```

Comment:
```suggestion
    <string name="confirmation_permission_upgrade">"You need to authorize this app to publish track recordings on OpenStreetMap. This requires to log in again. Proceed?"</string>
```

I found "will log you out of the app" to be misleading. Also made it a bit shorter.